actionpack/lib/action_controller/railtie.rb

```diff
@@ -67,5 +67,12 @@ class Railtie < Rails::Railtie #:nodoc:
         config.compile_methods! if config.respond_to?(:compile_methods!)
       end
     end
+
+    initializer "setup session store" do |app|
+      ActiveSupport.on_load(:action_controller) do
```

Comment:
I mean `railties` folder. This configuration have nothing to do with Action Controller. All work is being done in `Application::Configuration` class so this initializers should be in the `application.rb` file.